core/block_svg.js

```diff
@@ -523,14 +524,21 @@ Blockly.BlockSvg.prototype.tab = function(start, forward) {
 
 /**
  * Handle a mouse-down on an SVG block.
- * @param {!Event} e Mouse down event.
+ * @param {!Event} e Mouse down event or touch start event.
```

Comment:
What did you have in mind?

Things that don't make sense:
onTouchStart
onInteractionStart (maybe works, but is almost meaningless)
onGestureStart (already means something)
onDragStart (don't know it's a drag)
onSelect (already means something)